src/views/AddItem.jsx

```diff
@@ -1,3 +1,81 @@
+import { useState } from 'react';
+import { addItem } from '../api';
+import { useStateWithStorage } from '../utils';
+
+import './AddItem.css';
+
 export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+	const [itemToAdd, setItemToAdd] = useState({});
+	const [listToken, setListToken] = useStateWithStorage(
+		'my test list',
+		'tcl-shopping-list-token',
+	);
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		try {
+			const docRef = await addItem(listToken, {
+				itemName: itemToAdd.itemName,
+				daysUntilNextPurchase: itemToAdd.buyingFrequency,
+			});
+
+			docRef && alert(`${itemToAdd.itemName} was saved to the database`);
```

Comment:
I suggest resetting the form back to its original state (empty input and first radio button selected) when the form has been submitted. This week I was in an interview coding challenge and the interviewer pointed it out as a nice touch whenever you submit a React form.